### PR TITLE
feat: update readme with deploy process + target es2017 in tsc compiler

### PR DESCRIPTION
**Pull request by bertyhell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,28 @@ See postman collection (TODO add link)
...
 ### Troubleshooting
 
+## Deploy
+
+Steps to deploy:
+* update package.json version to match release branch version
+    * both the package.json in the root (important for the build pipeline)
+    * and the /server/package.json important to show the correct version in the express.js status route
+* merge release branch into master
+* add tag on master + push the tag (format: v1.1.1)
```
Comment (andrych) [inline README.md]: (tip: you can use npm version x.x.x | patch | minor | major)

File: server/tsconfig.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
 		"moduleResolution": "node",
 		"outDir": "dist",
 		"rootDir": ".",
-		"target": "es6",
+		"target": "es2017",
```
Comment (andrych) [inline server/tsconfig.json]: is het de bedoeling dat deze is veranderd?

File: README.md (hunk 1 above)
Comment (bertyhell) [reply README.md]: Beide package.json files moeten geupdate worden, dus npm version zal niet werken.

>   * both the package.json in the root (important for the build pipeline)
>   * and the /server/package.json important to show the correct version in the express.js status route

File: server/tsconfig.json (hunk 2 above)
Comment (bertyhell) [reply server/tsconfig.json]: Ja, read all about it here:
https://github.com/caolan/async/issues/1685

File: README.md (hunk 1 above)
Comment (andrych) [reply README.md]: ja klopt had ik niet aan gedacht

**Pull request merged**